src/main/java/com/znsio/teswiz/runner/Runner.java

```diff
@@ -37,7 +38,7 @@ public Runner() {
         throw new InvalidTestDataException("Required args not provided to Runner");
     }
 
-    public Runner(String configFilePath, String stepDefDirName, String featuresDirName) {
+    public Runner(String configFilePath, String stepDefDirName, String featuresDirName) throws IOException {
```

Comment:
why are we throwing exceptions? Throw a custom runtime exception

Reply:
Removed it got added because of method chaining, have fixed the method from where it was getting generated.